code/DebugBar.php

```diff
@@ -201,7 +201,7 @@ public static function includeRequirements()
         }
         // If jQuery is already included, set to false
         $js = Requirements::backend()->getJavascript();
-        foreach ($js as $url) {
+        foreach ($js as $url => $args) {
             $name = basename($url);
             if ($name == 'jquery.js' || $name == 'jquery.min.js') {
```

Comment:
Yeah, fair comment. Feel free to open an issue or a pull request to address this.